File: ws/fuentes/U3/WS28Taxes/Taxes/src/main/java/ec/edu/espe/taxes/controller/Tax.java
package ec.edu.espe.taxes.controller;

/**
 *
 * @author LABS-ESPE
 */
public class Tax {
    
    public float computeTax(float basePrice, float percentage, boolean charged){
        float ivaValue;
        
        if(!charged){
            ivaValue = 0.0F;
            return 0.0f;
        }else{
            ivaValue = basePrice*percentage/100;
            
        }
        return ivaValue;
    }
    
    public float computeIce(float basePrice){
        //CODE THE COMPUTATION OD ICE
        return 0.0F;
    }
    
}

File: hw/fuentes/U3/HW16UNITTEST/TestPy/Taxes/ec/edu/espe/taxes/controller/Tax.py
class Tax:
    def compute_tax(self, base_price: float, percentage: float, charged: bool) -> float:
        if not charged:
            return 0.0
        else:
            iva_value = base_price * percentage / 100
            return iva_value

    def compute_ice(self, base_price: float) -> float:
        # TODO: Implement ICE computation logic
        return 0.0
